test/mzworkflows_py/mzcompose.yml

```diff
@@ -6,17 +6,3 @@
 # As of the Change Date specified in that file, in accordance with
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
```

Comment:
The YAML file is now hollowed out. I will make it optional in a follow-up commit.